index.js

```diff
@@ -80,11 +78,12 @@ async function lintHtml(locale, data) {
   if (Array.isArray(data.value)) {
     data.value.forEach(v => {
       // Recursive magic...
-      lintHtml(locale, {name: `${v.name}[${v.variant}]`, value: v.value});
+      lintHtml(locale, {name: `${v.name}[${v.variant || v.attribute}]`, value: v.value});
     });
   } else {
     const results = await lint(data.value + "\n", {
-      "id-class-style": "dash"
+      "id-class-style": "dash",
+      "tag-bans": ["style", "i"]
```

Comment:
Turns out `<b>` is a banned tag by default: https://github.com/htmllint/htmllint/wiki/Options#tag-bans

🤷‍♂️